sql-cloudant/src/main/scala/org/apache/bahir/cloudant/common/ChangesRow.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2017 IBM Cloudant. All rights reserved.
```

Comment:
@emlaver -- is this an outdated copyright statement? If it is still valid you may need to check with the author and/or IBM if you can contribute this code (or variations of it) to open-source.

I am surprised the RAT check did not catch this.

CC @lresende